doc/source/conf.py

```diff
@@ -0,0 +1,94 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "TJPCov"
+copyright = "2019, LSST DESC"
```

Comment:
Should this be 2022 (or 2023)?

Reply:
No idea of what should go there. Maybe 2019-2023?

Reply:
I've added a (dynamic) range based on this: https://www.copyrightlaws.com/copyright-symbol-notice-year/